tests/Fixtures/e2e/Code_Coverage_Ignore/infection/coverage-xml/IgnoreClass.php.xml

```diff
@@ -1,83 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Looks like those files have been introduced by mistake in #720. In any case the updated `.gitignore` should deal with this and I double checked that I removed all of them